mdata/cache/accnt/accnt_test.go

```diff
@@ -0,0 +1,128 @@
+package accnt
+
+import (
+	"testing"
+	"time"
+)
+
+func getNew(maxSize int) *FlatAccnt {
+	a := &FlatAccnt{
+		total:       0,
+		metrics:     make(map[string]*FlatAccntMet),
+		maxSize:     uint64(maxSize),
+		lru:         NewLRU(),
+		evictQ:      make(chan *EvictTarget, 10),
```

Comment:
I didn't use  `NewFlatAccount()` because I wanted to be able to access it's private stuff like the ticker or the stats, but `NewFlatAccount()` returns an interface which doesn't have those.